### ISSUE TITLE
"Undefined symbols for architecture arm64" while integrating tvOS AppBoy Library

### ISSUE THREAD

**Issue by peruginni** [Appboy/appboy-ios-sdk#88]
Hello,

I have reused HelloSwift example and created new tvOS target. Then I have tried integrating tvOS Appboy Library, but I've got "Undefined symbols" error, when using "Appboy" class. 

Seems problematic only when integrating to Swift tvOS target (Objc example project works well on tvOS). Perhaps related to issues in #36 and #26  Do you have any idea how to resolve the error?

Podfile:

```
source 'https://github.com/CocoaPods/Specs.git'

target 'HelloSwift' do
    platform :ios, '8.0'
    use_frameworks!

    pod 'Appboy-iOS-SDK'
end

target 'HelloSwiftTVOS' do
    platform :tvos, '10.0'
    use_frameworks!

    pod 'Appboy-tvOS-SDK'
end

post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
        if target.name == "Appboy-iOS-SDK"
            config.build_settings["OTHER_LDFLAGS"] = '$(inherited) "-ObjC"'
        end
        if target.name == "Appboy-tvOS-SDK"
            config.build_settings["OTHER_LDFLAGS"] = '$(inherited) "-ObjC"'
        end
    end
  end
end
```

Bridging header

```
#import <Appboy_tvOS_SDK/AppboyKit.h>
```

In AppDelegate import...

```
import Appboy_tvOS_SDK
```

...and calling the class

```
Appboy.start(withApiKey: "6deb7788-edee-4a02-b75a-a254cdb9f58b", in:application, withLaunchOptions:launchOptions)
```

Complete forked project is on:
https://github.com/peruginni/appboy-ios-sdk

**Comment by briancaw** [comment]
Hi @peruginni 

Thanks very much for reporting this and for all of the great debugging info.  We can reproduce and we're actively investigating.  We'll provide more information ASAP.

Thanks,
Brian

**Comment by briancaw** [comment]
Hi @peruginni 

We're currently investigating what the correct approach is to solve this issue.  We have a fix that has worked in our testing, but we need to vet it further.  If you need an urgent fix, please let us know and we can provide more information about how to instrument that fix.  However, we'd like to inquire with the CocoaPods team about our approach before signing off.

Thanks,
Brian

**Comment by peruginni** [comment]
Hello @briancaw thanks for quick response and info about state! I was just exploring Appboy for possible future usage on tvOS. Urgent fix is not needed, I will wait for solution and then try again.

**Comment by Wenzhi** [comment]
Hey @peruginni,

We just released the tvOS SDK 2.24.3 with the fix. Could you try the new release and let us know whether it fixes the issue for you or not? 

Thanks,
Wenzhi

**Comment by briancaw** [comment]
Hi @peruginni 

Closing this for now - please let us know if you see any further issues here.  Thanks very much for reporting!
